src/common/marlin_server.cpp

```diff
@@ -107,22 +107,15 @@ msg_stack_t msg_stack = { '\0', 0 };
 
 void _add_status_msg(const char *const popup_msg) {
     char message[MSG_MAX_LENGTH];
-    size_t str_size = strlen(popup_msg);
-    if (str_size >= MSG_MAX_LENGTH - 1) {
-        strncpy(message, popup_msg, MSG_MAX_LENGTH - 1); // popup_msg is not always null-terminated...
-        message[MSG_MAX_LENGTH - 1] = '\0';
-    } else {
-        strncpy(message, popup_msg, str_size);
-        message[str_size] = '\0';
-    }
+    strlcpy(message, popup_msg, MSG_MAX_LENGTH);
```

Comment:
this won't zero the remaining bytes in message (strncpy does that) - change of behavior